x-pack/plugins/security_solution/public/detections/pages/alert_details/tabs/summary/cases_panel/index.tsx

```diff
@@ -31,13 +32,25 @@ import {
 } from '../translation';
 import { SummaryPanel } from '../wrappers';
 import { CasesPanelActions, CASES_PANEL_ACTIONS_CLASS } from './cases_panel_actions';
+import { RelatedCasesList } from './related_case';
 
 export interface CasesPanelProps {
   eventId: string;
   dataAsNestedObject: Ecs | null;
   detailsData: TimelineEventsDetailsItem[];
 }
 
+const StyledCasesFlexGroup = styled(EuiFlexGroup)`
+  max-height: 300px;
+  overflow-y: scroll;
```

Comment:
Please consider changing `scroll` to `auto` because `scroll` adds a permanent scroll width to the container event if there is not overflow. Please see below screenshot.

![Screenshot 2022-11-08 at 15 12 56](https://user-images.githubusercontent.com/7485038/200598971-1be3cd7e-79df-4f77-912d-ef56456ba31a.png)

Reply:
Will fix, thanks!